File: Scripts/newTask/json/perl/json.pl
use JSON;

for( $a=0; $a < 1000000; $a = $a + 1) {

my $data = decode_json('{ "foo": 1, "bar": [10, "apples"] }');

my $sample = { blue => [1,2], ocean => "water" };
my $json_string = encode_json($sample);
}

File: Scripts/newTask/json/go/json.go
package main

import "encoding/json"

func main() {

	for i := 0; i < 10000000; i++ {
    		var data interface{}
    		jsonString := json.Unmarshal([]byte(`{"foo":1, "bar":[10, "apples"]}`), &data)
		_ = jsonString
	}
}
